episodes/multi-dimensional-images.md

```diff
@@ -0,0 +1,634 @@
+
+![](fig/rgb-values.png){alt="A screenshot of an H+E slide of skin layers in 
+Napari, highlighting the (R,G,B) values"}
+
+We can see these different channels more clearly if we right click on the 'skin' 
+image layer and select:  
+`Split RGB`
+
+This shows the red, green and blue channels as separate image layers. Try 
+inspecting each one individually by clicking the ![](
+https://raw.githubusercontent.com/napari/napari/main/napari/resources/icons/visibility.svg
+){alt="A screenshot of Napari's eye button" height='30px'} icons to hide the 
+other layers.
+
+We can understand these RGB pixel values better by opening a different sample 
+image. Remove all layers, then select:  
+`File > Open Sample > napari builtins > Colorwheel (RGB)`
+
+![](fig/colorwheel-napari.png){alt="A screenshot of a colorwheel in Napari"}
+
+This image shows an RGB colourwheel - try hovering your mouse over different 
+areas, making note of the (R, G, B) values shown in the bottom left of Napari. 
+You should see that moving into the red area gives high values for R, and low 
+values for B and G. Equally, the green area shows high values for G, and the 
+blue area shows high values for B. The red, green and blue values are mixed 
+together to give the final colour.
+
+Recall from the [image display episode](image-display.md) that for most 
+microscopy images the colour of each pixel is determined by a colormap (or LUT). 
+Each pixel usually has a single value that is then mapped to a corresponding 
+colour via the colormap, and different colormaps can be used to highlight 
+different features. RGB images are different - here we have three values 
+(R, G, B) that unambiguously correspond to a colour for display. For example, 
+(255, 0, 0) would always be fully red, (0, 255, 0) would always be fully green 
+and so on... This is why there is no option to select a colormap in the layer 
+controls when an RGB image is displayed. In effect, the colormap is kindof 
+hard-coded into the image, with a full colour stored for every pixel location.
+
+These kinds of images are common when we are trying to replicate what can be 
+seen with the human eye. For example, photos taken with a standard camera or 
+phone will be RGB. They are less common in microscopy, although there are 
+certain research fields and types of microscope that commonly use RGB. A key 
+example is imaging of tissue sections for histology or pathology. You will also 
+often use RGB images when preparing figures for papers and presentations - RGB 
+images can be opened in all imaging software (not just scientific research 
+focused ones), so are useful when preparing images for display. As usual, always 
+make sure you keep a copy of your original raw data! 
+
+:::::::::::::::::::::::::::::::::::::: callout
+
+## Reading RGB in Napari
+
+How does Napari know an image is RGB? If an image's final dimension has a length 
+of 3 or 4, Napari will assume it is RGB and [display it as such
```

Comment:
I wasn't sure whether we should mention RGBA images here in case anyone is left wondering why the final dimension might be 4 instead of 3, but I think it's covered quite well in the hyperlink so I'd leave it as is.